src/User_Command.php

```diff
@@ -1197,4 +1197,41 @@ private function update_msuser_status( $user_ids, $pref, $value ) {
 		Utils\report_batch_operation_results( 'user', $verb, count( $user_ids ), $successes, $errors );
 	}
 
+	/**
+	 * Checks if given user credentials are valid or not.
```

Comment:
Actually think this would read better as `Checks if a user's credentials are valid or not.`